package.json

```diff
@@ -8,6 +8,11 @@
     "type": "git",
     "url": "git@github.com:satellytes/satellytes.com.git"
   },
+  "engines": {
+    "node": ">= 14.0.0",
+    "npm": "please-use-yarn",
```

Comment:
we used  that in another project to prevent people using npm to install things. But it's actually not always recognized so it's more like an information to read by humans at least.